frontend/src/store/reducers/app.ts

```diff
@@ -7,8 +7,18 @@ import {
 } from 'types/actions/app';
 import InitialValueTypes from 'types/reducer/app';
 
+const isDarkTheme = (): boolean => {
```

Comment:
also i see this function is duplicate under SideNav container can you please handle this one

Reply:
How about I shift these function to `api/theme` folder so that they can be reused later without code duplication.

Reply:
you can put this under `lib/theme`

Reply:
Cool!